packages/spec/integration/graphql-mock-server/package.json

```diff
@@ -6,7 +6,7 @@
     "node": "^10.13 || ^12.13"
   },
   "hops": {
-    "shouldPrefetchOnServer": false,
+    "shouldPrefetchOnServer": true,
```

Comment:
`shouldPrefetchOnServer` is deprecated, we renamed it to `allowServerSideDataFetching` 😬